flink-end-to-end-tests/test-scripts/test-runner-common.sh

```diff
@@ -93,7 +93,9 @@ function post_test_validation {
 
     if [[ ${exit_code} == 0 ]]; then
         cleanup
+        log_environment_info
     else
+        log_environment_info
```

Comment:
Why aren't you invoking this function outside of the `if` statement, if it applies to both cases?

Reply:
can't do it before since cleanup hasn't executed, can't do it after since we're doing an `exit` in one of the branches.

Reply:
True, logging the processes before the cleanup doesn't make much sense. Thanks for the explanation.